### PR TITLE
component test

### PR DESCRIPTION
**Pull request by AndronShermeta**
## Description
I wrote an enzyme test on the CategoryDelete page on the admin panel
All test passed

#### Screenshots

image

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### Checklist
- [✔] 🔽 My branch is up-to-date with "development" branch
- [✔] ✅ All tests passed locally
- [✔] ✨ My changes working with up-to-date front-end and back-end part locally, like a charm
- [✔] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}));
+
+describe('Categories test', () => {
+  let store;
+  jest.spyOn(reactRedux, 'useDispatch');
+  const mockHolder = jest.fn();
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(<CategoryDelete store={store} />);
```
Comment (dmytro07) [inline src/pages/categories/category-delete/test/category-delete.test.js:34]: Remove store

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  });
+  it('Should render Select', () => {
+    expect(wrapper.exists(Select)).toBe(true);
+  });
+  it('Should render StandardButton', () => {
+    expect(wrapper.exists(StandardButton)).toBe(true);
+  });
+  it('Button is working', () => {
+    useDispatch.mockReturnValue(mockHolder);
+    const wrapper = mount(<CategoryDelete store={store} />);
```
Comment (dmytro07) [inline src/pages/categories/category-delete/test/category-delete.test.js:61]: Remove mount

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: (selector) => selector(mockStore),
+  useDispatch: () => jest.fn()
+}));
+
+describe('Categories test', () => {
+  let store;
+  jest.spyOn(reactRedux, 'useDispatch');
```
Comment (dmytro07) [inline src/pages/categories/category-delete/test/category-delete.test.js:29]: Remove mock

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+const mockStore = store;
+
+jest.mock('react-redux', () => ({
+    ...jest.requireActual('react-redux'),
+    useSelector: (selector) => selector(mockStore),
+    useDispatch: () => jest.fn()
+  }));
+
+describe('Categories test', () => {
+    jest.spyOn(reactRedux, 'useDispatch');
```
Comment (yuliiakucher) [inline src/pages/categories/category-delete/test/category-delete.test.js:30]: please remove spyOn

**Review by dmytro07 — DISMISSED** [review]

**Comment by grebiniuk** [comment]
CI errors should be fixed

**Review by grebiniuk — CHANGES_REQUESTED** [review]
CI errors should be fixed

**Review by yuliiakucher — APPROVED** [review]

**Review by twihell — APPROVED** [review]

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Enzyme, { mount, shallow } from 'enzyme';
```
Comment (fulljsdev) [inline src/pages/categories/category-delete/test/category-delete.test.js:2]: shallow never used

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+    FormControl,
+    Grid,
+    InputLabel,
+    MenuItem,
```
Comment (fulljsdev) [inline src/pages/categories/category-delete/test/category-delete.test.js:9]: MenuItem never used

File: src/pages/categories/category-delete/test/category-delete.test.js [hunk 7]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Enzyme, { mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { useDispatch, useSelector } from 'react-redux';
```
Comment (fulljsdev) [inline src/pages/categories/category-delete/test/category-delete.test.js:4]: useSelector never used

**Review by fulljsdev — COMMENTED** [review]
remove unused imports

**Review by grebiniuk — APPROVED** [review]

**Pull request merged**